### PR TITLE
Default the GitHub Token input

### PR DESCRIPTION
**Pull request by chrispat**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ runs:
 inputs:
   token:
     description: 'GitHub token'
+    default: ${{ secrets.GITHUB_TOKEN }}
```
Comment (tcbyrd) [inline action.yml:9]: Ah nice. Does this means we can completely remove the parameter from the workflow yaml?

Comment (chrispat) [reply action.yml:9]: Yes we shouldn't need to include it.

Comment (tcbyrd) [reply action.yml:9]: @chrispat This doesn't appear to be working with `secrets.GITHUB_TOKEN`: https://github.com/paper-spa/pages/runs/4172669926?check_suite_focus=true

```log
Error: /home/runner/work/pages/pages/./action.yml (Line: 19, Col: 14):
Error: /home/runner/work/pages/pages/./action.yml (Line: 19, Col: 14): Unrecognized named-value: 'secrets'. Located at position 1 within expression: secrets.GITHUB_TOKEN
Error: Fail to load /home/runner/work/pages/pages/./action.yml
```

Looks like `secrets` may not be loaded yet when evaluating what to pass for the default input?

Comment (chrispat) [reply action.yml]: Sorry I put the wrong thing.  Fixed now

Comment (tcbyrd) [reply action.yml]: Wonder if we should add some documentation around this. This is what comes up when you google that error: https://github.community/t/unrecognized-named-value-secrets-in-action-yml/207729

**Review by tcbyrd — APPROVED** [review]
It works!

**Pull request merged**